src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -59,6 +66,16 @@ class GroupTagValues extends AsyncComponent<
       property('count')
     ).reverse();
 
+    if (sortedTagValueList.length === 0 && this.props.environments?.length > 0) {
```

Comment:
Can we destructure `environments` from `this.props` in the above block? https://github.com/getsentry/sentry/pull/18728/files#diff-95e6c286d2d062e433b73a138ffd114fR59-R62

Prop types suggests `environments` is required, do we need optional chaining?

Reply:
Thanks, good call out 👍